### PR TITLE
Fixed multiple toggles due to repeated code

### PR DESCRIPTION
**Pull request by darshkpatel**
some remotes repeat the code within 200ms thus toggling the relay twice , this fixes the toggle function.

### CODE REVIEW

File: code/espurna/ir.ino [hunk 1]
```diff
@@ -14,7 +14,7 @@ Copyright (C) 2017-2018 by François Déchery
 
 IRrecv * _ir_recv;
 decode_results _ir_results;
-
+unsigned long last = millis();
```
Comment (mcspr) [inline code/espurna/ir.ino]: `last` is global variable for the whole espurna codebase. notice how other ones have `_ir` prefix

File: code/espurna/ir.ino [hunk 2]
```diff
@@ -43,7 +43,12 @@ void _irProcessCode(unsigned long code) {
                 relayStatus(0, button_value);
             }
             if (button_mode == IR_BUTTON_MODE_TOGGLE) {
+		    if (millis() - last > 250){
```
Comment (mcspr) [inline code/espurna/ir.ino]: Please use spaces instead of tabs.

File: code/espurna/ir.ino [hunk 3]
```diff
@@ -43,7 +43,12 @@ void _irProcessCode(unsigned long code) {
...
                 relayToggle(button_value);
```
Comment (mcspr) [inline code/espurna/ir.ino]: Shouldn't `last` be updated here? `now - last > 250` is always true

**Comment by darshkpatel** [comment]
apologies for missing obvious things.

**Comment by darshkpatel** [comment]
@mcspr  ping

**Comment by xoseperez** [comment]
Thank you!

**Pull request merged**